padd.sh

```diff
@@ -543,17 +543,19 @@ GetVersionInformation() {
   # PADD version information...
```

Comment:
```suggestion
  # PADD version information...
  padd_version_latest="$(curl --silent https://api.github.com/repos/pi-hole/PADD/releases/latest | grep '"tag_name":' | awk -F \" '{print $4}')"
  padd_version_latest_converted="$(VersionConverter "${padd_version_latest}")"
  # is PADD up-to-date?
  if [ -z "${padd_version_latest}" ]; then
    padd_version_heatmap=${yellow_text}
  else
    padd_version_converted=$(VersionConverter "${padd_version}")

    if [ "${padd_version_converted}" -lt "${padd_version_latest_converted}" ]; then
      padd_out_of_date_flag="true"
      padd_version_heatmap=${red_text}
    else
      # local and remote PADD version match or local is newer
      padd_version_heatmap=${green_text}
    fi
  fi
```

Calling a function or a shell inside an if block is a lint warning.

Reply:
Sure. Was there a reason for avoiding the conversion outside?